<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_profil</class>
 <widget class="QDialog" name="gestion_profil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
font-family:century gothic;
background:#ffb56a;
font-size:17px;

}
QGroupBox
{
background:#ffb56a;
background-image:url(:/selem2.jpg);
}
#ajouter
{
background:#299a7a;
border-radius:15px;
}
#ajouter:hover
{
color:#333;
border-radius:60px;
background:red;
}
#pushButton_supprimer
{
background:#299a7a;
border-radius:15px;
}
#pushButton_supprimer:hover
{
color:#333;
border-radius:60px;
background:red;
}
#pushButton_modifier
{
background:#299a7a;
border-radius:15px;
}
#pushButton_modifier:hover
{
color:#333;
border-radius:60px;
background:red;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>AJOUTER</string>
    </attribute>
    <widget class="QPushButton" name="ajouter">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>360</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>AJOUTER</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>140</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID_PROFIL</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditidprofil">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>140</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditposteprofil">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>240</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>240</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>POSTE</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>SUPPRIMER</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_supp_profil">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>230</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_supprimer">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>350</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SUPPRIMER</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>230</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID EMPLOYER :</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>AFFICHER</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>50</y>
       <width>231</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>MODIFIER</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_modifier">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>390</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MODIFIER</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>170</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string> ID PROFIL :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>270</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NOUVELLE POSTE :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditidprofil1">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>170</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditposteprofil1">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>270</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
